index.html

```diff
@@ -151,13 +151,18 @@ <h2>Scope</h2>
             <a href="https://www.w3.org/TR/did-spec-registries/">DID Specification Registries</a>, including transitioning it from a WG Note to a W3C Registry, should such a formal registry be defined by W3C Process and should that process allow such a transition.
           </li>
           <li>
-            <a href="https://www.w3.org/TR/did-rubric/">DID Method Rubric</a> <i class=todo>(Note: this link is currently 404, the planned publication for this note is mid-August)</i>
+            <a href="https://w3c.github.io/did-rubric/">DID Method Rubric</a>, including transitioning it from a WG Note to a W3C Registry, should such a formal registry be defined by W3C Process and should that process allow such a transition. 
```

Comment:
```suggestion
            <a href="https://w3c.github.io/did-rubric/">DID Method Rubric</a>, including transitioning it from a WG Note to a W3C Registry, should such a formal registry be defined by W3C Process, should that process allow such a transition, and should the previous working group define the registration process for such a registry.
```

Reply:
@jandrieu @iherman would this language work, do you think?

Reply:
I am fine with such a change. An alternative is to replace the whole reference with the text in §2.2:

>  If, during the chartered period of the Working Group, the next version of the W3C Process is adopted, incorporating the concept of a Registry Track, the Working Group will consider publishing the DID Method Rubric as a Registry Track document instead of a Working Group Note.

which less committal than what is in §1 and does not get too much into the details.